stable/keycloak/templates/statefulset.yaml

```diff
@@ -21,6 +21,9 @@ spec:
         app: {{ template "keycloak.name" . }}
         release: "{{ .Release.Name }}"
     spec:
+    {{- with .Values.keycloak.serviceAccountName }}
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for RBAC best practices and default serviceAccountName if not defined in values.yaml